tests/queries/custom_aggregate/query.trickle

```diff
@@ -0,0 +1,22 @@
+define tumbling window by_two
+with
+  size = 2
+end;
+
+mod m with
```

Comment:
being in a model makes it look like the aggregate is defined in tremor-script scope, which somewhat doesn't make a lot of sense if it can't be used in a tremor-script context in that case I think it would be better at trickle level